packages/curvenote-cli/src/submissions/submit.utils.ts

```diff
@@ -61,41 +87,123 @@ export function venueQuestion(session: ISession) {
   };
 }
 
-export async function determineSubmissionKind(
+export async function determineCollectionAndKind(
   session: ISession,
   venue: string,
-  opts?: { kind?: string },
+  collections: CollectionsDTO,
+  opts?: { kind?: string; collection?: string },
 ) {
-  let kinds;
-  try {
-    kinds = await getFromJournals(session, `sites/${venue}/kinds`);
-  } catch (err: any) {
+  const openCollections = collections.items.filter((c) => c.open);
+  if (openCollections.length === 0) {
+    session.log.info(
+      `${chalk.red(`⛔️ no collections are open for submissions at venue "${venue}"`)}`,
+    );
+    process.exit(1);
+  }
+
+  session.log.debug(`Explicit collection provided: ${opts?.collection}`);
```

Comment:
Minor stuff, but this should probably only be logged this way if `opts.collection` is defined.